src/frontend/src/views/Data/cellRenderers.tsx

```diff
@@ -137,8 +157,8 @@ const SAMPLE_CUSTOM_RENDERERS: Record<string | number, CellRenderer> = {
               <StyledChip
                 data-test-id="row-sample-status"
                 size="small"
-                label={label.label}
-                status={label.status}
+                label={label()?.label}
+                status={label()?.status}
```

Comment:
Can this ever be undefined/null?  It looks like `label()` would just return the default case if qcMetrics is null.  If typescript is mad, adding a return type to the function may help.